content/blog/2019-01-15-waterinfo-tidal-eel.md

```diff
@@ -0,0 +1,173 @@
+---
+slug: "waterinfo-tidal-eel"
+title: wateRinfo - Downloading tidal data to understand the behaviour of a migrating eel
+authors:
+  - Stijn Van Hoey
+  - Peter Desmet
+date: 2019-01-15
+categories: blog
+topicid:
+tags:
+  - software-peer-review
+  - r
+  - package
+  - water
+  - data-access
+  - tidal-data
+  - fishes
+  - movement-data
+  - acoustic-telemetry
+---
+
+> Do you know what that sound is, Highness? Those are the Shrieking Eels — if you don't believe me, just wait. They always grow louder when they're about to feed on human flesh. If you swim back now, I promise, no harm will come to you. I doubt you will get such an offer from the Eels.
+>
+> *Vizzini, [The Princess Bride](https://youtu.be/KGcat9tGZVU)*
+
+European eels ([*Anguilla anguilla*](https://en.wikipedia.org/wiki/Anguilla_anguilla)) have it tough. Not only are they depicted as monsters in movies, they are [critically endangered](https://doi.org/10.2305/IUCN.UK.2014-1.RLTS.T60344A45833138.en) in real life. One of the many aspects that is contributing to their decline is the reduced connectivity between their freshwater and marine habitats. Eels are catadromous: they live in freshwater, but migrate to the Sargasso Sea to spawn, a route that is blocked by numerous human structures (shipping locks, sluices, pumping stations, etc.). [Pieterjan Verhelst](http://www.marinebiology.ugent.be/node/68491) studies the impact of these structures on the behaviour of eels, making use of the [fish acoustic receiver network](http://lifewatch.be/en/fish-acoustic-receiver-network) that was established as part of the Belgian LifeWatch observatory. This animated video gives a quick introduction to his research and the receiver network:
+
+[![Tagging research on eel in Belgium](https://img.youtube.com/vi/7YQVgl3QPyY/0.jpg)](https://www.youtube.com/watch?v=7YQVgl3QPyY)
```

Comment:
Can you add some of this credit text for the video? "Tagging research on eel in Belgium. Animated video. Coordination: Karen Rappé and Pieterjan Verhelst, Animation Steve Bridger, Voice: Bryan Kopta (EN), Production: LifeWatch Belgium, Executive producer: VLIZ Oostende -  Belgium."

Reply:
Good point, I'll add this to the acknowledgments section.

Reply:
@peterdesmet can you have a quick check on the adapted acknowledgements?

Reply:
Updated.